docs/reference/esql/functions/max.asciidoc

```diff
@@ -1,7 +1,24 @@
 [discrete]
 [[esql-agg-max]]
 === `MAX`
-The maximum value of a numeric field.
+
+*Syntax*
+
+[source,esql]
+----
+MAX(column)
+----
+
+*Parameters*
+
+`column`::
+Column from which to return the maximum value.
```

Comment:
This does not have to be a column - it can be an expression.

Reply:
I've got a PR brewing that updates the docs for the aggregate functions, now that they support inline expressions. It'll be fixed in that PR.